### PR TITLE
Finished dom-creation assignment

### PR DESCRIPTION
**Pull request by DreSean-Sutton**
_C__Users_Dre%20Sean%20Sutton_repos_c0122-code-solutions_dom-creation_index html

### CODE REVIEW

File: dom-creation/main.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+    $columnThirdDiv.classList.add('column-third');
+    $pokemonCardDiv.classList.add('pokemon-card');
+    $pokemonTextDiv.classList.add('pokemon-card-text');
+    $pokemonImage.setAttribute('src', pokemon[i].imageUrl);
+    $PokemonName.textContent = pokemon[i].name;
+    $PokemonNumber.textContent = pokemon[i].number;
+    $PokemonText.textContent = pokemon[i].description;
+
+    $rowDiv.appendChild($columnThirdDiv);
```
Comment (thebearingedge) [inline dom-creation/main.js]: Please review the instructions again, this function should not add anything to the page.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: dom-creation/main.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  {
+    number: '009',
+    name: 'Blastoise',
+    description: 'It crushes its foe under its heavy body to cause fainting. In a pinch, it will withdraw inside its shell.',
+    imageUrl: 'images/blastoise.png'
+  }
+];
+
+function renderPokemon(pokemon) {
+  for (var i = 0; i < pokemon.length; i++) {
```
Comment (thebearingedge) [inline dom-creation/main.js]: This function should take one object and return one dom tree, no loop.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Sorry for the spam, my first comment was not included in the initial review

**Comment by DreSean-Sutton** [comment]
_C__Users_Dre%20Sean%20Sutton_repos_c0122-code-solutions_dom-creation_index html

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**